rtf

```diff
@@ -16,11 +16,13 @@ Usage:
     rtf scenario list
     rtf scenario env <platform>
     rtf scenario run <platform> <scenario> [--no-finally] [--filter=<test1>,<test2>,...] [--format=<format>]
+    rtf test from-rule <platform> <uuid> <test_name> [--with-scenario]
 
 Options:
     --no-finally       Do not run tests tagged FINALLY in the scenario, to avoid coming back to initial state
     --filter=...       Only run provided test list from scenario
     --format=<format>  Output format to use (progress, documentation, html or json) [Default: documentation]
+    --with-scenario    Generate a sample scenario file to use this test independantly
```

Comment:
Not sure if the option name is easy to understand 

what do you think of "--create-scenario" or "--create" ?